packages/manager/apps/dedicated/client/app/account/user/user.html

```diff
@@ -39,6 +39,20 @@
                 ><span data-translate="user_account_advanced_title"></span>
             </oui-header-tabs-item>
         </oui-header-tabs>
+        <oui-guide-menu
+            data-text="{{:: 'user_account_guides_header' | translate }}"
+        >
+            <oui-guide-menu-item
+                data-ng-repeat="guide in $ctrl.guides track by $index"
+                data-href="{{:: guide.url }}"
+                data-on-click="$ctrl.trackClick(guide.tracking)"
+                data-external
+            >
+                <span
+                    data-translate="{{:: 'user_account_guides_header_' + guide.key }}"
```

Comment:
i think in this case the `data-ng-bind` is more relevant

Reply:
It's same.